auth0/resource_auth0_connection.go

```diff
@@ -243,6 +218,7 @@ var connectionSchema = map[string]*schema.Schema{
 				},
 				"allowed_audiences": {
 					Type:        schema.TypeSet,
+					Computed:    true,
```

Comment:
The issue here is that we define all possible parameters of the options object for all the different connection types and with the legacy SDK (v1) if we don't set something up for a TypeSet initially it will be read as "" but then after a state update it will be automatically set to an empty []string making the tests fail cuz there's an unexpected plan change. To fix this we need to set computed on this property and several others.